src/js/utils/ErrorMessageUtil.js

```diff
@@ -0,0 +1,41 @@
+const MESSAGE_VARIABLE = /\{\{([^}]+)\}\}/g;
```

Comment:
Please change the pattern to `/\{\{([^{}]+)\}\}/g` to properly handle additional curly braces (e.g `{{{variable}}}`)